<?xml version="1.0" encoding="UTF-8"?>
<robot generated="20161205 10:38:49.282" generator="Robot 3.0 (Python 2.7.11 on win32)">
<suite source="C:\Users\Mark.Wang\Documents\robotframework\LoRaWANScripts\Project\testcase\yl\yl_group_testing.robot" id="s1" name="Yl Group Testing">
<test id="s1-t1" name="Create Dictionary Test">
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>hello wolrd</arg>
</arguments>
<status status="PASS" endtime="20161205 10:38:49.397" starttime="20161205 10:38:49.397"></status>
</kw>
<status status="PASS" endtime="20161205 10:38:49.397" critical="yes" starttime="20161205 10:38:49.396"></status>
</test>
<status status="PASS" endtime="20161205 10:38:49.398" starttime="20161205 10:38:49.284"></status>
</suite>
<statistics>
<total>
<stat fail="0" pass="1">Critical Tests</stat>
<stat fail="0" pass="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat fail="0" id="s1" name="Yl Group Testing" pass="1">Yl Group Testing</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20161205 10:38:49.391" level="ERROR">Error in file 'C:\Users\Mark.Wang\Documents\robotframework\LoRaWANScripts\Project\testcase\yl\yl_group_testing.robot': Importing test library 'C:\Users\Mark.Wang\Documents\robotframework\LoRaWANScripts\Project\Resources\mylibrary.py' failed: ImportError: No module named mylibrary
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Windows\system32\python27.zip
  c:\python27\DLLs
  c:\python27\lib
  c:\python27\lib\plat-win
  c:\python27\lib\lib-tk
  c:\python27
  c:\python27\lib\site-packages
  c:\python27\lib\site-packages\wx-2.8-msw-unicode</msg>
</errors>
</robot>
